src/main/scala/chatapp/Chat.scala

```diff
@@ -33,14 +35,14 @@ class Chat(var messages: List[ChatMessage]) extends RdObject {
 		//TODO: terminate
 	}
 
-	override def currentVersion(): Long = {
+	def currentVersion(): Long = {
```

Comment:
Should versions be defined on the RDO rather than the implementation of the RDO?